packages/app/screens/feed/create group form/CreateGroupFormScreen2.js

```diff
@@ -0,0 +1,164 @@
+      handleInputValidationScreen2();
+    }, [category, tags, location, website]);
+  
+    function handleInputValidationScreen2() {
+      let isValid = true;
+      const websiteRegex =
+        /^(?!(https?:\/\/)([\da-z\.-]+)\.([a-z\.]{2,6})([\/\w \.-]*)*\/?$).+$/i;
+  
+      if (category.trim() === ``) {
+        setCategoryError(`Please select a category`);
+        isValid = false;
+      } else {
+        setCategoryError(``);
+      }
+  
+      if (tags.length < 3) {
+        setTagsError(`Please select at least 3 tags`);
+        isValid = false;
+      } else {
+        setTagsError(``);
+      }
+  
+      if (website.trim() === ``) {
+        setWebsiteError(``);
+      } else if (websiteRegex.test(website)) {
+        setWebsiteError(`Invalid Website Link`);
+        isValid = false;
+      } else {
+        setWebsiteError(``);
+      }
+  
+      return isValid;
+    }
+  
+    return (
+      <ScrollView
+        behavior={Platform.OS === "ios" ? "padding" : "height"}
+        style={styles.container}
+        showsVerticalScrollIndicator={false}
+        keyboardShouldPersistTaps="handled">
+        <TouchableWithoutFeedback onPress={Keyboard.dismiss}>
+          <View style={styles.inner}>
+            <Button
+              onPress={() => navigation.navigate("Create Group Form 1")}
+              title="BACK"
+            />
+  
+            <View>
+              <Text style={styles.header}>
+                Category<Text style={styles.important}>* {categoryError} </Text>
+              </Text>
+              <Dropdown
+                options={categoryOptions}
+                value={category}
+                setValue={setCategory}
+              />
+            </View>
+  
+            <View>
+              <Text style={styles.header}>
+                Tags<Text style={styles.important}>* {tagsError} </Text>
+              </Text>
+              <TagInput
+                value={inputValue}
+                setValue={setInputValue}
+                tags={tags}
+                setTags={setTags}
+                maxTags={10}
+              />
+            </View>
+  
+            <View>
+              <Text style={styles.header}>
+                Location
+                <Text
+                  value={website}
+                  style={styles.important}
+                  onChangeText={(newText) => setLocation(newText)}>
+                  {" "}
+                  {locationError}{" "}
+                </Text>
+              </Text>
+              <TextInput placeholder="Username" style={styles.input} />
+            </View>
+  
+            <View>
+              <Text style={styles.header}>
+                Website<Text style={styles.important}> {websiteError} </Text>
+              </Text>
+              <TextInput
+                value={website}
+                placeholder="https://example.com"
+                onChangeText={(newText) => setWebsite(newText)}
+                style={styles.input}
+              />
```

Comment:
For any website, the user shouldn't need to type "https://". Maybe add it in advance?